### DISCUSSION TITLE
[Feature] 使用GUI界面下载本子

### DISCUSSION THREAD

**Discussion by Einzieg** [hect0x7/JMComic-Crawler-Python#388] [Ideas]
### 新功能请求

打包成pc端可执行exe，使用gui界面进行配置以及下载本子

**Comment by hect0x7** [comment]
这个功能也有点太大😂

**Comment by Shua-github** [comment]
我做过。。。但是代码被家长删了，现在做了api

**Comment by Shua-github** [comment]
> > 我做过。。。但是代码被家长删了，现在做了api
> 
> 为什么不上传GitHub上？

因为不会上传

**Comment by Shua-github** [comment]
> > > 我做过。。。但是代码被家长删了，现在做了api
> > 
> > 
> > 为什么不上传GitHub上？
> 
> 因为不会上传

现在有jm客户端支持下载本子，仓库：https://github.com/tonquer/JMComic-qt

**Comment by OverFlame** [comment]
> 这个功能也有点太大😂

今晚我在自己电脑上配置完基础功能后在尝试用d指导来写一个GUI，写完以后我贴在这里，作者您看一下可以吗

**Comment by OverFlame** [comment]
> 这个功能也有点太大😂

其他plugings的配置方法拷打d指导仍然有许多bug，不过这个基础的可以更改下载目录的GUI做好了
code引用有点问题，我以txt形式上传了

[JMD_TEST.txt](https://github.com/user-attachments/files/19441206/JMD_TEST.txt)

**Comment by hect0x7** [comment]
> > 这个功能也有点太大😂
> 
> 其他plugings的配置方法拷打d指导仍然有许多bug，不过这个基础的可以更改下载目录的GUI做好了 code引用有点问题，我以txt形式上传了
> 
> [JMD_TEST.txt](https://github.com/user-attachments/files/19441206/JMD_TEST.txt)

我觉得你没必要发到这里，可以自己建一个仓库，作为一个项目去维护

**Comment by OverFlame** [comment]
> > > 这个功能也有点太大😂
> > 
> > 
> > 其他plugings的配置方法拷打d指导仍然有许多bug，不过这个基础的可以更改下载目录的GUI做好了 code引用有点问题，我以txt形式上传了
> > [JMD_TEST.txt](https://github.com/user-attachments/files/19441206/JMD_TEST.txt)
> 
> 我觉得你没必要发到这里，可以自己建一个仓库，作为一个项目去维护

诶我没有这方面经验啊（摊手）
而且我现在感觉没有这个实力（目移）
试试看罢

**Comment by Mmmtttt** [comment]
我的项目可以完美实现哦，超级好用，功能还不止这一点，支持禁漫、哔咔、javdb、javbus、missav、jable等多个平台的内容导入和管理，喜欢的话各位大佬帮忙给个星，感谢~❤
https://github.com/Mmmtttt/ULTIMATE_WEB
